### DISCUSSION TITLE
Overwrite trace flag in traceparent header

### DISCUSSION THREAD

**Discussion by wesbrownfavor** [istio/istio#47041] [Q&A]
Hey folks,

We have problems with third parties unexpectedly sending us [traceparents](https://www.w3.org/TR/trace-context/#traceparent-header-field-values) with the [traceflag](https://www.w3.org/TR/trace-context/#trace-flags) set to 00 instead of 01.  As a result Istio won't send traces for that request to our telemetry software.

Is there a way to override the traceflag at the Ingress Gateway level or prevent the traceparent from being used for external requests?

**Comment by zirain** [comment]
can you try to override header with VirtualService?

**Reply by wesbrownfavor** [reply]
I can drop the header with the virtual service but then I don't get trace data between my ingress gateway and my application's pod.
